checkov/terraform/checks/resource/aws/IAMUserNotUsedForAccess.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
+
+from typing import Any
+
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class IAMUserNotUsedForAccess(BaseResourceCheck):
+    def __init__(self) -> None:
+        name = "Ensure IAM users are not defined " \
```

Comment:
This is great detail, but too long for a title. How about:

`Ensure access is controlled through SSO not AWS IAM defined users`

We can add the rest of the detail to the docs associated with the check.

Reply:
Thank you for the suggestion. This change has been implemented. Please find the previous comment below for the docs. 

Ensure IAM users are not defined. AWS Access should be controlled by roles via an SSO, inline with AWS security best practices. Require human users to use federation with an identity provider to access AWS using temporary credentials.